packages/components/nodes/documentloaders/Cheerio/Cheerio.ts

```diff
@@ -127,13 +128,22 @@ class Cheerio_DocumentLoaders implements INode {
             if (!limit) limit = '10'
             else if (parseInt(limit) < 0) throw new Error('Limit cannot be less than 0')
             const pages: string[] =
-                relativeLinksMethod === 'webCrawl' ? await webCrawl(url, parseInt(limit)) : await xmlScrape(url, parseInt(limit))
+                selectedLinks && selectedLinks.length > 0
+                    ? selectedLinks.slice(0, parseInt(limit))
+                    : relativeLinksMethod === 'webCrawl'
+                    ? await webCrawl(url, parseInt(limit))
+                    : await xmlScrape(url, parseInt(limit))
             if (process.env.DEBUG === 'true') console.info(`pages: ${JSON.stringify(pages)}, length: ${pages.length}`)
```

Comment:
Can we use the logger instead of console for DEBUG ?

Reply:
Updated the code to use `logger` instead of `console`